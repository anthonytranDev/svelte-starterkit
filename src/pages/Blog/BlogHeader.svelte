<style>
  .blog-description {
    color: #686868;
    font-style: italic;
  }
</style>

<div data-testid="BlogHeader">
  <h1>Blog</h1>
  <p class="blog-description">Latest news</p>
</div>
