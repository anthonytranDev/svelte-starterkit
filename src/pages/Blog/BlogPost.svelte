<div data-testid="BlogPost">
  <h2>Blog Post #{$$props.params.id}</h2>
</div>
