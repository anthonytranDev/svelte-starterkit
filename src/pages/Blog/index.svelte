<script>
  import BlogHeader from "./BlogHeader.svelte"
</script>

<BlogHeader />
