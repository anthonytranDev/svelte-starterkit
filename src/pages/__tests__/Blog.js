import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/svelte'
import { Router } from 'svelte-routing'

import Blog from '../Blog/index.svelte'

describe('Blog', () => {
  it('should render', () => {
    const { getByTestId } = render(
      <Router>
        <Blog />
      </Router>
    )

    expect(getByTestId('Blog')).toMatchSnapshot()
  })
})
