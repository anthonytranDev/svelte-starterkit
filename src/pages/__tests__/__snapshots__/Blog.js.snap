// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog should render 1`] = `
<div
  data-testid="Blog"
>
  <div
    data-testid="BlogHeader"
  >
    <h1>
      Blog
    </h1>
     
    <p
      class="blog-description svelte-y06stb"
    >
      Latest news
    </p>
  </div>
   
  <div>
    <a
      href="/blog/1"
    >
      Blog Post #1
    </a>
  </div>
   
  <div>
    <a
      href="/blog/2"
    >
      Blog Post #2
    </a>
  </div>
</div>
`;
